### PR TITLE
final

### PR DESCRIPTION
**Pull request by NicholasTruson**

### CODE REVIEW

File: js/index.js [hunk 1]
```diff
@@ -38,5 +38,106 @@ const siteContent = {
...
+navA[1].text = 'Product'
+navA[2].setAttribute('test', '.nav-item-3')
+navA[2].text = 'Vision'
+navA[3].setAttribute('test', '.nav-item-4')
+navA[3].text = 'Features'
+navA[4].setAttribute('test', '.nav-item-5')
+navA[4].text = 'About'
+navA[5].setAttribute('test', '.nav-item-6')
+navA[5].text = 'Contact'
+
```
Comment (rleslie1015) [inline js/index.js]: Here is an example of doing it the way we talked about in stand up using the forEach method. This code loops through the nodelist you have selected and assigns values from the siteContent object.

```
nav.forEach(function(value, i){
  value.textContent=siteContent.nav[`nav-item-${i+1}`];
});
```

File: js/index.js [hunk 2]
```diff
@@ -38,5 +38,106 @@ const siteContent = {
...
+// Header
+
+let mainHeader = document.querySelector('h1')
+mainHeader.textContent = siteContent['cta']['h1']
+
```
Comment (rleslie1015) [inline js/index.js]: Great job creating a variable for the h1!

File: js/index.js [hunk 3]
```diff
@@ -38,5 +38,106 @@ const siteContent = {
...
+let footer = document.querySelector('footer p')
+
+footer.textContent = siteContent ["footer"]["copyright"]
+
+// Navigation Color Change
+
+navA.forEach(link => link.style.color = "green")
+
+// New Navigation Items
+
```
Comment (rleslie1015) [inline js/index.js]: Love these comments they are so helpful keep that up!

**Review by thereactgirl — COMMENTED** [review]
Awesome job you met all the objectives!!

**Pull request merged**